docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 version: '3'
+name: koku
```

Comment:
Can you link to docs that describe what this field is for?

Why are we changing all of the service names in this file?

Reply:
> Can you link to docs that describe what this field is for?

The [name top-level argument](https://docs.docker.com/compose/compose-file/#name-top-level-element) sets the prefix used for each container name The docs don't explain that very well. The default value is the name of the directory container the compose file.

I mainly added this for explicit consistency. It can be omitted, though, since we explicitly set the `container_name` for almost every service.

Reply:
> Why are we changing all of the service names in this file?

For consistency and to reduce some typing.

Some services had the compose project name prefixed, such as `koku-server`, others did not, such as `db` and `services-client`. Removing the prefix from the service names makes compose commands more succinct.

`docker compose up server masu-server worker db trino`

Naming all the containers with a consistent prefix makes it clear which project a collection of containers is associated with when running more than one collection of containers.

```
> docker ps
CONTAINER ID   NAMES                 IMAGE                                           STATE     architecture   PORTS
0ef4c6415f53   sources-api-server    sources-api                                     running   aarch64        0.0.0.0:9394->9394/tcp, 0.0.0.0:3000->8000/tcp
cb71b8245e91   sources-redis         redis:5.0.4                                     running                  0.0.0.0:6378->6379/tcp
24ff6f08e1b9   sources-kafka         confluentinc/cp-kafka                           running   arm64          0.0.0.0:9092->9092/tcp, 0.0.0.0:29092->29092/tcp
0dbc7abd0f7a   sources-db            postgres:14                                     running                  0.0.0.0:15434->5432/tcp
c74090343132   sources-zookeeper     confluentinc/cp-zookeeper                       running   arm64          2181/tcp, 2888/tcp, 3888/tcp
7cfcb87eedca   koku-worker-1         koku_base                                       running   aarch64        8000/tcp, 0.0.0.0:6001->9000/tcp
f62f3c96a9f7   koku-server           koku_base                                       running   aarch64        0.0.0.0:8000->8000/tcp, 0.0.0.0:8001->9000/tcp
50e199bb9a33   masu-server           koku_base                                       running   aarch64        0.0.0.0:5042->8000/tcp, 0.0.0.0:5001->9000/tcp
217bea790e0d   koku-hive-metastore   quay.io/mskarbek/ubi-hive:3.1.2-metastore-008   running   arm64          1000/tcp, 0.0.0.0:9083->8000/tcp
c9e68d2d0b1f   koku-minio            local-minio:latest                              running   x86_64         0.0.0.0:9000->9000/tcp, 0.0.0.0:9090->9090/tcp
46e9236c4530   koku-redis            redis:5.0.4                                     running                  0.0.0.0:6379->6379/tcp
fb618c665975   koku-unleash          unleashorg/unleash-server:3.9                   running                  0.0.0.0:4242->4242/tcp
dcdd08127c18   koku-db               postgres:13                                     running                  0.0.0.0:15432->5432/tcp
```

Part of me thinks that `masu-server` should maybe be `koku-masu-server`.